tests/setup.log

```diff
@@ -0,0 +1,134 @@
+- Proxy which can point the dynamic kruize service (ports can be changing)
+- UI should communicate with the proxy (fixed ip and port)
+- UI should read env and figure out the fixed proxy endpoint and port
+
+- API forwarder fixed ip and port
+
+API server - http for UI and grpc for internal endpoint
+
+Every request from UI or curl will go to API server, this will figure out kruize url's and port and then talk via GRPC and respond to curl or UI via http
+
+[
+  {
+    "version": "1.0",
```

Comment:
This needs to be v2.0 ?

Reply:
I have skipped it as it's a log file. Will change it as well.

Reply:
Done.